### PR TITLE
People 178 add tooltips to btns

### PR DESCRIPTION
**Pull request by DariuszMusielak**
# Ticket
https://netguru.atlassian.net/browse/PEOPLE-178

# Description
- Added tooltip container - it's required since there is no way to add tooltip on disabled btn
- Trigger tooltips on page ready
- Added tooltips for buttons in skills flow 

### Before merge checklist
- [ ] CircleCI is passing
- [ ] Proper labels are set

### CODE REVIEW

File: app/views/draft_skills/index.html.haml [hunk 1]
```diff
@@ -40,11 +40,13 @@
...
+                      = link_to draft_skill, class: "btn btn-info", role: 'button' do
+                        %i.glyphicon.glyphicon-search
                     - unless draft_skill.resolved?
-                      = link_to edit_draft_skill_path(draft_skill),
-                                class: "btn btn-primary #{update_request_btn_class(draft_skill, current_user)}",
-                                role: 'button' do
-                        %i.glyphicon.glyphicon-edit
+                      .tooltip-container{ data: { toggle: 'tooltip', placement: 'top' }, title: 'Show request details for change' }
+                        = link_to edit_draft_skill_path(draft_skill),
+                          class: "btn btn-primary #{update_request_btn_class(draft_skill, current_user)}",
```
Comment (netguru-hound) [inline app/views/draft_skills/index.html.haml]: Line is too long. [106/80]

File: app/views/draft_skills/index.html.haml [hunk 2]
```diff
@@ -40,11 +40,13 @@
...
+                  .skills__actions
+                    .tooltip-container{ data: { toggle: 'tooltip', placement: 'top' }, title: 'Display request details for change' }
+                      = link_to draft_skill, class: "btn btn-info", role: 'button' do
+                        %i.glyphicon.glyphicon-search
                     - unless draft_skill.resolved?
-                      = link_to edit_draft_skill_path(draft_skill),
-                                class: "btn btn-primary #{update_request_btn_class(draft_skill, current_user)}",
-                                role: 'button' do
-                        %i.glyphicon.glyphicon-edit
+                      .tooltip-container{ data: { toggle: 'tooltip', placement: 'top' }, title: 'Show request details for change' }
```
Comment (netguru-hound) [inline app/views/draft_skills/index.html.haml]: Line is too long. [131/80]

File: app/views/draft_skills/index.html.haml [hunk 3]
```diff
@@ -40,11 +40,13 @@
                     = link_to draft_skill.reviewer&.full_name, user_path(draft_skill.reviewer)
                 %td= draft_skill.reviewer_explanation
                 %td
-                  .btn-group.skills__actions{ role: "group", aria: { label: "Actions" } }
-                    = link_to draft_skill, class: "btn btn-info", role: 'button' do
-                      %i.glyphicon.glyphicon-search
+                  .skills__actions
+                    .tooltip-container{ data: { toggle: 'tooltip', placement: 'top' }, title: 'Display request details for change' }
+                      = link_to draft_skill, class: "btn btn-info", role: 'button' do
```
Comment (netguru-hound) [inline app/views/draft_skills/index.html.haml]: Line is too long. [85/80]

File: app/views/draft_skills/index.html.haml [hunk 4]
```diff
@@ -40,11 +40,13 @@
                     = link_to draft_skill.reviewer&.full_name, user_path(draft_skill.reviewer)
                 %td= draft_skill.reviewer_explanation
                 %td
-                  .btn-group.skills__actions{ role: "group", aria: { label: "Actions" } }
-                    = link_to draft_skill, class: "btn btn-info", role: 'button' do
-                      %i.glyphicon.glyphicon-search
+                  .skills__actions
+                    .tooltip-container{ data: { toggle: 'tooltip', placement: 'top' }, title: 'Display request details for change' }
```
Comment (netguru-hound) [inline app/views/draft_skills/index.html.haml]: Line is too long. [132/80]

File: app/views/skills/index.html.haml [hunk 5]
```diff
@@ -30,20 +30,24 @@
...
+                        = link_to edit_draft_skill_path(skill.requested_change),
+                          class: "btn btn-danger #{update_request_btn_class(skill.requested_change, current_user)}",
+                          role: 'button' do
+                          %i.glyphicon.glyphicon-comment
                     - else
-                      = link_to edit_skill_path(skill),
-                                class: "btn btn-primary",
-                                role: 'button' do
-                        %i.glyphicon.glyphicon-cog
+                      .tooltip-container{ data: { toggle: 'tooltip', placement: 'top' }, title: 'Click to edit skill' }
```
Comment (netguru-hound) [inline app/views/skills/index.html.haml]: Line is too long. [139/80]<br>Line contains trailing whitespace

File: app/views/skills/index.html.haml [hunk 6]
```diff
@@ -30,20 +30,24 @@
...
-                                role: 'button' do
-                        %i.glyphicon.glyphicon-comment
+                      .tooltip-container{ data: { toggle: 'tooltip', placement: 'top' }, title: 'Access denied because there already is request for change' }
+                        = link_to edit_skill_path(skill),
+                          class: 'btn btn-primary disabled',
+                          role: 'button' do
+                          %i.glyphicon.glyphicon-cog
+                      .tooltip-container{ data: { toggle: 'tooltip', placement: 'top' }, title: 'Review request details for change' }                          
+                        = link_to edit_draft_skill_path(skill.requested_change),
+                          class: "btn btn-danger #{update_request_btn_class(skill.requested_change, current_user)}",
```
Comment (netguru-hound) [inline app/views/skills/index.html.haml]: Line is too long. [116/80]

File: app/views/skills/index.html.haml [hunk 7]
```diff
@@ -30,20 +30,24 @@
...
-                      = link_to edit_draft_skill_path(skill.requested_change),
-                                class: "btn btn-danger #{update_request_btn_class(skill.requested_change, current_user)}",
-                                role: 'button' do
-                        %i.glyphicon.glyphicon-comment
+                      .tooltip-container{ data: { toggle: 'tooltip', placement: 'top' }, title: 'Access denied because there already is request for change' }
+                        = link_to edit_skill_path(skill),
+                          class: 'btn btn-primary disabled',
+                          role: 'button' do
+                          %i.glyphicon.glyphicon-cog
+                      .tooltip-container{ data: { toggle: 'tooltip', placement: 'top' }, title: 'Review request details for change' }
```
Comment (netguru-hound) [inline app/views/skills/index.html.haml]: Line is too long. [159/80]<br>Line contains trailing whitespace

File: app/views/skills/index.html.haml [hunk 8]
```diff
@@ -30,20 +30,24 @@
...
                     - if skill.requested_change
-                      = link_to edit_skill_path(skill),
-                                class: "btn btn-primary disabled",
-                                role: 'button' do
-                        %i.glyphicon.glyphicon-cog
-                      = link_to edit_draft_skill_path(skill.requested_change),
-                                class: "btn btn-danger #{update_request_btn_class(skill.requested_change, current_user)}",
-                                role: 'button' do
-                        %i.glyphicon.glyphicon-comment
+                      .tooltip-container{ data: { toggle: 'tooltip', placement: 'top' }, title: 'Access denied because there already is request for change' }
```
Comment (netguru-hound) [inline app/views/skills/index.html.haml]: Line is too long. [157/80]

File: app/views/skills/index.html.haml [hunk 9]
```diff
@@ -30,20 +30,24 @@
                 %td= skill.description
                 %td= skill.rate_type
                 %td
-                  .btn-group.skills__actions{ role: "group", aria: { label: "Actions" } }
-                    = link_to skill, class: "btn btn-info", role: 'button' do
-                      %i.glyphicon.glyphicon-search
+                  .skills__actions
+                    .tooltip-container{ data: { toggle: 'tooltip', placement: 'top' }, title: 'Display skill details' }
```
Comment (netguru-hound) [inline app/views/skills/index.html.haml]: Line is too long. [119/80]

File: app/views/skills/_last_5_draft_skills.html.haml [hunk 10]
```diff
@@ -23,11 +23,13 @@
...
+                = link_to draft_skill, class: "btn btn-info", role: 'button' do
+                  %i.glyphicon.glyphicon-search
               - unless draft_skill.resolved?
-                = link_to edit_draft_skill_path(draft_skill),
-                          class: "btn btn-primary",
-                          role: 'button' do
-                  %i.glyphicon.glyphicon-edit
+                .tooltip-container{ data: { toggle: 'tooltip', placement: 'top' }, title: 'Review request details for change' }
+                  = link_to edit_draft_skill_path(draft_skill),
+                            class: "btn btn-primary #{update_request_btn_class(draft_skill, current_user)}",
```
Comment (netguru-hound) [inline app/views/skills/_last_5_draft_skills.html.haml]: Line is too long. [108/80]

File: app/views/skills/_last_5_draft_skills.html.haml [hunk 11]
```diff
@@ -23,11 +23,13 @@
...
+            .skills__actions
+              .tooltip-container{ data: { toggle: 'tooltip', placement: 'top' }, title: 'Display request details for change' }
+                = link_to draft_skill, class: "btn btn-info", role: 'button' do
+                  %i.glyphicon.glyphicon-search
               - unless draft_skill.resolved?
-                = link_to edit_draft_skill_path(draft_skill),
-                          class: "btn btn-primary",
-                          role: 'button' do
-                  %i.glyphicon.glyphicon-edit
+                .tooltip-container{ data: { toggle: 'tooltip', placement: 'top' }, title: 'Review request details for change' }
```
Comment (netguru-hound) [inline app/views/skills/_last_5_draft_skills.html.haml]: Line is too long. [127/80]

File: app/views/skills/_last_5_draft_skills.html.haml [hunk 12]
```diff
@@ -23,11 +23,13 @@
             - else
               %span.text-danger not resolved
           %th
-            .btn-group.skills__actions{ role: "group", aria: { label: "Actions" } }
-              = link_to draft_skill, class: "btn btn-info", role: 'button' do
-                %i.glyphicon.glyphicon-search
+            .skills__actions
+              .tooltip-container{ data: { toggle: 'tooltip', placement: 'top' }, title: 'Display request details for change' }
```
Comment (netguru-hound) [inline app/views/skills/_last_5_draft_skills.html.haml]: Line is too long. [126/80]

File: app/views/notifications/index.html.haml [hunk 13]
```diff
@@ -35,14 +35,14 @@
                   - else
                     %span.text-danger Not marked as read
                 %td
-                  .btn-group.skills__actions{ role: "group", aria: { label: "Actions" } }
+                  .tooltip-container{ data: { toggle: 'tooltip', placement: 'top' }, title: 'Display notification details' }
```
Comment (netguru-hound) [inline app/views/notifications/index.html.haml]: Line is too long. [124/80]

**Comment by marcinjak** [comment]
LGTM :+1: (Circle fixed in other PR)

**Pull request merged**